src/services/searchSkylink.js

```diff
@@ -80,20 +80,6 @@ export default async function searchSkylink(input) {
     }
   }
 
-  // any arbitrary url
-  if (input.startsWith("https://")) {
-    const skylink = await requestSkylink(input);
-
-    if (skylink) return skylink;
-  }
-
-  if (input.match(/^[^./]+$/)) {
```

Comment:
This was a bit hard to follow, the entire function actually. Since I added the HEAD response check as a fallback I figured it was fine to drop this all together. We explicitly check for hns URLs above using the `HNS_DOMAIN_MATCHER` regex.

Reply:
I didn't really understand how this regex relates to an hns URL. Which is implied since it's followed up with `const address = await skynetClient.getHnsUrl(input);`

Reply:
This is to match any string without dots and try to see if it matches hns domain. This allows searches like "uniswap" or "skyfeed".

Reply:
ok, brought it back in that case and added a comment